x-pack/plugins/security_solution/public/notes/components/add_note.tsx

```diff
@@ -87,7 +87,7 @@ export const AddNote = memo(
         createNote({
           note: {
             timelineId: timelineId || '',
-            eventId,
+            eventId: eventId || '',
```

Comment:
when we save a note associated to a timeline only. I changed this behavior because I don't really want to have a scenario where the `eventId` property is not present, as I don't know (yet) how to filter for that, I only know how to filter for values being empty or not 😆 

```
nodeBuilder.is(`${noteSavedObjectType}.attributes.eventId`,'');
```